src/core/cubic.c

```diff
@@ -404,13 +446,86 @@ CubicCongestionControlOnDataAcknowledged(
         goto Exit;
     }
 
+    //
+    // Update HyStart++ RTT sample.
+    //
+    if (AckEvent->MinRttValid) {
+        uint32_t Eta;
+
+        if (Cubic->HyStartState != HYSTART_DONE) {
+            //
+            // Update Min RTT for the first N ACKs.
+            //
+            if (Cubic->HyStartAckCount < QUIC_HYSTART_DEFAULT_N_SAMPLING) {
+                Cubic->MinRttInCurrentRound =
+                    CXPLAT_MIN(
+                        Cubic->MinRttInCurrentRound,
+                        AckEvent->MinRtt);
+                Cubic->HyStartAckCount++;
+            } else {
+                Eta =
+                    CXPLAT_MIN(
+                        QUIC_HYSTART_DEFAULT_MAX_ETA,
+                        CXPLAT_MAX(
+                            QUIC_HYSTART_DEFAULT_MIN_ETA,
+                            Cubic->MinRttInLastRound / 8));
```

Comment:
Where does the `8` come from?

Reply:
Should this be `QUIC_HYSTART_DEFAULT_N_SAMPLING` right?

Reply:
Nope. It's just another constant value.